client/src/main/java/io/obswebsocket/community/client/message/event/Event.java

```diff
@@ -54,15 +56,21 @@
 
 @Getter
 @ToString(callSuper = true)
-public abstract class Event extends Message {
+public abstract class Event<T> extends Message {
+  @ToString.Exclude private final transient Intent eventIntent;
```

Comment:
The intent is part of the messageData so including it will mean it is in there twice.
I'm not entirely sure if I refactored the need of this field away already, initially I used a setData to set the data but because this is now in the constructor, I don't think the field is needed anymore.

Reply:
Can't it just stay in message data then?

Reply:
It was in Message initially, it's just that v5 protocol evolved over time and this got moved to message data

Reply:
It can be done like [this](https://github.com/obs-websocket-community-projects/obs-websocket-java/pull/135/commits/5cf69cb94fd3bf7690463e4802ec6541af8b0de4). But creating a 'dummy' messageData only to put 
the intent in there so that it can be taken out feels a bit messy.

I think I'm mostly not really in favor of the computeEventSubscription method of getting the intent from the event.

Reply:
I'm trying to contact you so we can have a little chat about this so we can think of what the the options, etc...
Here is the [Discord server invitation](https://discord.gg/XmNsZBbf)

Reply:
Let's keep it like it is in the mean time